src/main/resources/db/migration/V1_20__update_region_table.sql

```diff
@@ -0,0 +1,53 @@
+--Delete building locations and court venues
+TRUNCATE court_venue CASCADE;
+TRUNCATE building_location CASCADE;
```

Comment:
Will this not fail your test case https://github.com/hmcts/rd-location-ref-api/blob/master/src/functionalTest/java/uk/gov/hmcts/reform/lrdapi/RetrieveBuildingLocationDetailsFunctionalTest.java#L45 as there will be no data when you truncate?